data/json/overmap/overmap_land_use_codes.json

```diff
@@ -129,7 +129,7 @@
     "id": "wetland_saltwater",
     "land_use_code": 14,
     "name": "Saltwater Wetland",
-    "detailed_definition": "DEP Wetlands (1:12,000) WETCODEs 11 and 27.",
+    "detailed_definition": "DEP Wetlands (1:12,000) WETCODEs 11, and 27.",
```

Comment:
Tdere's only two elements here, not 3+, so serial commas wouldn't apply.

Reply:
I must have got carried away, as I didn't even mark them in Transifex. Either way fixed, thanks!